### PR TITLE
remove styles npm/cdn bento packages

### PR DESCRIPTION
**Pull request by kvchari**
removes css from being included in the autogenerated npm/cdn packages

### CODE REVIEW

**Review by dethstrobe — APPROVED** [review]

File: build-system/tasks/extension-helpers.js [hunk 1]
```diff
@@ -778,14 +778,8 @@ async function generateBentoEntryPointSource(name, toExport, options) {
 
   return dedent(`
     import {BaseElement} from '../base-element';
-    
+
     function defineElement() {
-      const css = __css__;
```
Comment (estherkim) [inline build-system/tasks/extension-helpers.js:783]: Shouldn't `__css__` and all its references be deleted as well then?

Comment (kvchari) [reply build-system/tasks/extension-helpers.js:783]: ah forgot to remove the rest of the references - updated

**Comment by estherkim** [comment]
LGTM pending bento e2e tests

**Comment by kvchari** [comment]
fixed e2e tests

**Pull request merged**